docs/content.zh/docs/ops/metrics.md

```diff
@@ -1642,7 +1642,7 @@ logged by `SystemResourcesMetricsInitializer` during the startup.
   </tbody>
 </table>
 
-## Latency tracking
+## End-to-End record travelling latency tracking
 
 Flink allows to track the latency of records travelling through the system. This feature is disabled by default.
```

Comment:
```suggestion
Flink allows tracking the latency of records travelling through the system. This feature is disabled by default.
```

Reply:
**same for English version**

Reply:
I think this should be okay and this also existed in many places within Flink's docs.